common/types/message/message.go

```diff
@@ -183,6 +183,12 @@ type ChunkInfo struct {
 	TxBytes       []byte      `json:"tx_bytes"`
 }
 
+// SubCircuitRowUsage tracing info added in v0.11.0rc8
+type SubCircuitRowUsage struct {
```

Comment:
we can replace this with l2geth's type.

Reply:
I prefer current way.  First it is a super simple struct.  second Now ccc row usage and prover row usage are same type, in future we may have different details. So keeping 2 structs is not bad.

Reply:
makes sense.